capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module

```diff
@@ -164,9 +168,19 @@ function c4m_restful_quick_post_c4m_location_field_entity_property_info(&$info,
  */
 function c4m_restful_quick_post_preprocess_html(&$vars) {
   $item = menu_get_item();
+  $allowed_paths = array(
+    'node/add',
+    'node/%/edit',
+    'node',
+  );
 
-  if (!menu_get_object() && isset($item['map']) && $item['map'][0] != 'node') {
-    // Not a node page and not a node/add(edit) page.
+  if (!in_array($item['path'], $allowed_paths)) {
+    // Not a group dashboard page and not a node/add(edit) page.
+    return;
+  }
+  if (drupal_is_front_page() && !og_context()) {
```

Comment:
You need to refactor the code more:
* If we ever decide to disable the /node page (and we should).
* If we ever decide that we use another default home page (e.g. /home).

Then the c4m-app will no longer load on the group dashboard because the path (/home) no longer in the allowed_paths list.

Maybe you need to change the allowed paths list like this:
```
$allowed_paths = array(
  'node/add',
  'node/%/edit',
  variable_get('site_frontpage', 'node')
);
```

Should the add path not be like:
```
  'node/add/%'
```

We don't need the app on `node/add` itself as this is an overview of all possible content type…